sdk/nodejs/Makefile

```diff
@@ -28,7 +28,8 @@ build_package:: ensure
 	cp tests/runtime/jsClosureCases_8.js bin/tests/runtime
 	cp tests/runtime/jsClosureCases_10_4.js bin/tests/runtime
 	cp -R tests/automation/data/. bin/tests/automation/data/
-	cp README.md ../../LICENSE package.json ./dist/* bin/
+	cp README.md ../../LICENSE ./dist/* bin/
+	cp package.json yarn.lock bin/ && cd bin && yarn remove mockpackage && rm yarn.lock
```

Comment:
Why do we copy `yarn.lock` and then remove it?

Reply:
In order to run `yarn remove mockpackage` to remove mockpackage from `package.json` for the bin/, I put a `yarn.lock` file in the same directory. The alternative is that I find an obscure flag in yarn that ignores a yarn.lock. The reason it's being removed is because it wasn't there in the first place and I wanted to avoid adding anything that wasn't in there in the first place.

I preferred using `yarn` to remove it because it seemed safer than sed/awk/grep/etc and trying to match text.